packages/app/src/vis-packs/core/hooks.ts

```diff
@@ -73,18 +80,19 @@ const useBounds = createMemo(getBounds);
 const useValidDomainForScale = createMemo(getValidDomainForScale);
 
 export function useDomain(
-  valuesArray: NdArray<number[]> | number[],
+  valuesArray: AnyNumArray,
```

Comment:
You'll see it further down, but this type resolves to `NdArray<TypedArray | number[]> | TypedArray | number[]`. So it's really permissive.